arch/lkl/include/asm/thread_info.h

```diff
@@ -13,16 +13,16 @@ typedef struct {
 } mm_segment_t;
 
 struct thread_info {
+	struct lkl_jmp_buf sched_jb;
 	struct task_struct *task;
+	struct task_struct *prev_sched;
+	struct lkl_sem *sched_sem;
 	unsigned long flags;
-	int preempt_count;
+	unsigned long stackend;
 	mm_segment_t addr_limit;
-	struct lkl_sem *sched_sem;
-	struct lkl_jmp_buf sched_jb;
-	bool dead;
 	lkl_thread_t tid;
-	struct task_struct *prev_sched;
-	unsigned long stackend;
+	int preempt_count;
+	bool dead;
```

Comment:
Are these changes needed? If so can you please pull them out in a separate commit with a description about why they are needed?

Also AFAIR stackend should be the last entry in the structure.

Reply:
Not needed, it was originally to minimize gcc padding by ordering members in descending order. Will remove it. Thanks !